walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,41 @@
 
 package com.google.sps;
 
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
+import java.util.Collections;
 
 public final class FindMeetingQuery {
```

Comment:
I don't know if I am commenting too much.

Reply:
I think its fine as long as ur explaining why youre doing something, instead of just stating the obvious. But there definitely be a comment about the class right above the "public final class Blah"